content/agent/faq/how-do-i-uninstall-the-agent.md

```diff
@@ -0,0 +1,112 @@
+---
+title: How do I uninstall the agent?
+kind: faq
+further_reading:
+- link: "/agent/"
+  tag: "Documentation"
+  text: Learn more about the Datadog Agent
+---
+
+## Agent v6
+### Amazon Linux/Debian/Fedora/Ubuntu
+```
+$ sudo apt-get --purge remove datadog-agent -y
+```
+
+### CentOS
+
+* For CentOS 5:
+
+    ```
+    $ sudo yum remove datadog-agent-base
+    ```
+
+* For CentOS 6:
+
+    ```
+    $ sudo yum remove datadog-agent
+    ```
+
+### Mac OS
+
+Stop and close the Datadog Agent: via the bone icon in the Tray.
+
+Drag the Datadog Application from the application folder to the Trash Bin.
+
+```
+$ sudo rm -rf /opt/datadog-agent
+$ sudo rm -rf /usr/local/bin/datadog-agent
+$ sudo rm -rf ~/.datadog-agent/**​ #to remove broken symlinks
+```
+If you ran the optional install commands to have the Agent run at boot time, run the following to finish uninstalling:
```

Comment:
Should the `Agent` here be capitalized? I ask because in the PR, when Datadog proceeds it, Agent is capitalized, but when Datadog does not proceed it, it usually is not.

Reply:
Well, Agent should be always capitalized.. so it's a mistake, pushing a fix.